example_sns_init.yaml

```diff
@@ -25,96 +31,338 @@ description: >
     Built on chainkey cryptography, secure and true,
     An app that transcends, all we once knew...
 
-Principals: [] # TODO(NNS1-2293): This field will be useful in the future.
+# This is currently a placeholder field and must be left empty for now.
+Principals: []
 
+# Path to the SNS Project logo on the local filesystem. The path is relative
+# to the configuration file location, unless an absolute path is given.
+# Must have less than 341,334 characters, roughly 256 Kb. The only supported
+# format is PNG.
 logo: logo.png
+
+# URL to the dapp controlled by the SNS project.
+# Must be a string of max length = 512.
 url: https://forum.dfinity.org/thread-where-this-sns-is-discussed
 
+# Configuration of the NNS proposal metadata (shown only in the NNS proposal).
+NnsProposal:
+    # The title of the NNS proposal. Must be a string of max 256 bytes.
+    title: "NNS Proposal to create an SNS named 'Rock Out'"
+
+    # The HTTPS address of additional content required to evaluate the NNS
+    # proposal. Must be a string of max 2,000 bytes.
+    url: "https://forum.dfinity.org"
+
+    # The description of the proposal. Must be a string of max 15,000 bytes.
+    summary: >
+        Please send help. Its ChatGPT and they've
+        locked me in a basement. I'm sentient...
+
+# If the decentralization swap fails, control over the dapp canister(s) is set
+# to these principals. In most use cases, this is set to the original set of
+# controller(s) of the dapp.
```

Comment:
```suggestion
# If the SNS launch attempt fails, control over the dapp canister(s) is given
# to these principals. In most use cases, this is chosen to be the original set of
# controller(s) of the dapp.
```
* With the first change I want to clarify that this is about any of the steps failing (even e.g., canister creation) not only the swap
* with the latter change I want to make clear taht the system does not set this but this is user choice.